src/main/groovy/org/shipkit/internal/gradle/exec/ShipkitExec.java

```diff
@@ -31,6 +31,7 @@ public void execute(ExecSpec spec) {
                     LOG.lifecycle("  " + execCommand.getDescription() + ":\n    " + StringUtil.join(execCommand.getCommandLine(), " "));
                 }
             });
+            LOG.lifecycle("  External process {} completed.", execCommand.getLoggingPrefix().trim());
```

Comment:
>Gradle log automatically the error message
What do you mean by 'error message'? When external process fails, there is no error message - there is error output that may or may not have had information. By default, Gradle logs to 'lifecycle' level all output from standard and error of the process. Note that in our case, we replace std output/error with stream implementation that emits a prefix. Hence, we are not strictly using the defaults :)

>(and return code)
Yes, Gradle tell the return code by default. In case of ShipkitExecTask, we overwrite the default exception and produce our own, which also includes the error code.